### PR TITLE
Updated current args available for passing in docs

### PR DESCRIPTION
**Pull request by Omeryl**

### CODE REVIEW

File: doc/self-hosted/index.md [hunk 1]
```diff
@@ -38,18 +38,20 @@ USAGE
   $ code-server [WORKDIR]
 
 ARGUMENTS
-  WORKDIR  [default: (directory to binary)] Specify working dir
+  WORKDIR  [default: /root/project] Specify working dir
```
Comment (kylecarbs) [inline doc/self-hosted/index.md]: This is only the default for the docker container.

Comment (Omeryl) [reply doc/self-hosted/index.md]: Yeah I just noticed it, I misread my diff, fixing it now.

Comment (Omeryl) [reply doc/self-hosted/index.md]: Resolved with a force-push to my branch.

**Comment by Omeryl** [comment]
@kylecarbs I've amended the commit and fixed my branch, apologies for the typo.

**Pull request merged**